--- dylib-unhell.py	2022-07-09 08:19:23.000000000 +0800
+++ dylib-unhell-patched.py	2022-07-13 12:54:52.000000000 +0800
@@ -10,11 +10,15 @@
 sys_re = re.compile("^/System")
 usr_re = re.compile("^/usr/lib/")
 exe_re = re.compile("@executable_path")
+loader_re = re.compile("@loader_path")
+rpath_re = re.compile("@rpath")
 
 def is_user_lib(objfile, libname):
     return not sys_re.match(libname) and \
            not usr_re.match(libname) and \
            not exe_re.match(libname) and \
+           not loader_re.match(libname) and \
+           not rpath_re.match(libname) and \
            not "libobjc." in libname and \
            not "libSystem." in libname and \
            not "libc." in libname and \
