From 0d9d4adb0e09c2e60efedae2f38e4873dbe4b59e Mon Sep 17 00:00:00 2001
From: eko5624 <562433403@qq.com>
Date: Sun, 30 Apr 2023 20:04:59 +0800
Subject: [PATCH] Mac: fix double icons on dock

---
 thumbfast.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thumbfast.lua b/thumbfast.lua
index 6521300..a6401ec 100644
--- a/thumbfast.lua
+++ b/thumbfast.lua
@@ -374,7 +374,7 @@ local function spawn(time)
     has_vid = vid or 0
 
     local args = {
-        mpv_path, path, "--no-config", "--msg-level=all=no", "--idle", "--pause", "--keep-open=always", "--really-quiet", "--no-terminal",
+        "mpv", path, "--no-config", "--msg-level=all=no", "--idle", "--pause", "--keep-open=always", "--really-quiet", "--no-terminal",
         "--edition="..(mp.get_property_number("edition") or "auto"), "--vid="..(vid or "auto"), "--no-sub", "--no-audio",
         "--start="..time, "--hr-seek=no",
         "--ytdl-format=worst", "--demuxer-readahead-secs=0", "--demuxer-max-bytes=128KiB",
